src/main/java/com/lightbend/rp/LagomApp.java

```diff
@@ -42,7 +42,7 @@ private void parseServices(String serviceJson) {
         for(int i = 0; i < obj.length(); ++i) {
             JSONObject o = obj.getJSONObject(i);
 
-            Endpoints.Endpoint e = endpoints.addEndpoint(o.getString("name"), "http");
+            Endpoints.Endpoint e = endpoints.addEndpoint(0, o.getString("name"), "http");
```

Comment:
JFYI this will insert all the services in reverse order (keep inserting at index 0 and pushing the rest down).